frontend/sportsmatch-app/src/components/Match.tsx

```diff
@@ -8,23 +8,42 @@ import {
   LuSettings2,
 } from 'react-icons/lu'
 import { Link } from 'react-router-dom'
+import { EventDTO } from '../generated/api/models/EventDTO'
+import { useEffect, useState } from 'react'
+import { ExSecuredEndpointService, OpenAPI, UserDTO } from '../generated/api'
 
 interface InProgressProps {
-  event: {
-    id: number
-    maxElo: number
-    minElo: number
-    dateEnd: string
-    dateStart: string
-    location: string
-    title: string
-    sport: string
-    playerOne: string
-    playerTwo?: string
-  }
+  event: EventDTO
```

Comment:
could we delete this InProgressProps and just pass in the eventDTO?

Reply:
we can, but we will have to pass each property of EventDTO to Match component (see image) if we pass the eventDTO like this
` function InProgress(event: EventDTO) { ....`
![image](https://github.com/MatejFrnka/sportsmatch/assets/131284389/5252c212-27bc-48ca-be00-ad1f23cec5c1)